framework/platform/bpl/include/bpl_cfg.h

```diff
@@ -349,6 +363,15 @@ int bpl_cfg_get_dfs_reentry();
  */
 int bpl_cfg_get_passive_mode();
 
+/**
+ * Returns DCS channel pool [sec].
```

Comment:
Copy paste error - should be `Returns the DCS (Dynamic Channel Selection) parameters)` or something like that..

Reply:
same here.
all BPL_DCS_PARAMS related code can be removed .
Its unused and not related to DCS feature we are currently working on.

Reply:
done